modules/RTC/ScreenObtainer.js

```diff
@@ -163,9 +163,18 @@ const ScreenObtainer = {
             getDisplayMedia = navigator.mediaDevices.getDisplayMedia.bind(navigator.mediaDevices);
         }
 
+        const { disableAGC, disableAP, enableHdAudio } = this.options;
+        const audioProcessingValue = !disableAGC && !disableAP;
```

Comment:
Why are you checking for AP and AGC here? If the HD audio option is selected then they need to be false, period.